source/utilities.tex

```diff
@@ -835,6 +835,11 @@
 \begin{codeblock}
 !is_convertible_v<U1, T1> || !is_convertible_v<U2, T2>
 \end{codeblock}
+This constructor is defined as deleted if
```

Comment:
```suggestion
\pnum
This constructor is defined as deleted if
```
(Probably needs a new subclause number, I think)

Reply:
An alternate is to have something along the lines of
```
Remarks:
  -- The expression inside explicit is equivalent to:
!is_convertible_v<U1, T1> || !is_convertible_v<U2, T2>
  -- This constructor is defined as deleted if reference_constructs_from_temporary_v<first_type, U1&&> is true or reference_constructs_from_temporary_v<second_type, U2&&> is true.
```

Reply:
We don't necessarily need a paragraph number just because we start a new line after "codeblock". I'd like to omit the `\pnum` here to highlight it's part of the preceding `\remarks`.

Reply:
> We don't necessarily need a paragraph number just because we start a new line after "codeblock". I'd like to omit the `\pnum` here to highlight it's part of the preceding `\remarks`.

I should note that it seems you accepted some of those `\pnum` at #5276.

Reply:
Noted. I guess we're globally inconsistent in that area. No action; I'll leave this item open in case the Project Editor wishes to do something about it.

Reply:
Thanks, but this strikes me as another false drive to a lexical consistency that is not helpful: we make paragraphs when paragraphs are useful, not based on some low-level lexical condition. A codeblock can be part of a paragraph, or it can be its own paragraph, whatever suits the demands of the text.